### PR TITLE
Renamed doc/ folder to docs/

### PR DESCRIPTION
**Pull request by StraToN**
In order to use the doc/ folder for Github Pages, one needs to rename it docs/.

### CODE REVIEW

**Comment by fleskesvor** [comment]
Looks good to me. :sunglasses:

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Installation
+------------
+
+[Git submodules](git_submodule.md)
+
+Reference
+---------
+
+[Background](background.md)
```
Comment (fleskesvor) [inline docs/index.md]: There should be empty lines between these links so that they are not inlined.

Comment (fleskesvor) [reply docs/index.md]: The same goes for the Scripting and Tools sections.

Comment (StraToN) [reply docs/index.md]: Fixed

**Pull request merged**